test/bench.cpp

```diff
@@ -4,87 +4,370 @@
 #include <string>
 #include <vector>
 
+#include <sstream>
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+#include <cstdio>
+#include <ctype.h>
+
+#define MAX(X, Y) (((X) > (Y)) ? (X) : (Y))
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+
+// Define screen dimensions
+#define SCREEN_WIDTH    1200
+#define SCREEN_HEIGHT   600
+
+// Define params of input squares
+#define INPUT_WIDTH   60
+#define INPUT_HEIGHT   45
+
+#define FONT_PATH   "../assets/ttf/DejaVuSansMono.ttf"
+
 using namespace lorina;
 
-struct bench_statistics
-{
-  uint32_t number_of_inputs = 0;
-  uint32_t number_of_outputs = 0;
-  uint32_t number_of_dffs = 0;
+struct bench_statistics {
+    uint32_t number_of_inputs = 0;
+    uint32_t number_of_outputs = 0;
+    uint32_t number_of_dffs = 0;
```

Comment:
Formatting for the sake of formatting is not allowed: follow the style in the file as close as reasonable while editing.